### DISCUSSION TITLE
Allow reorder of layers via drag (grab handles?)

### DISCUSSION THREAD

**Discussion by jimkring** [ni/labview-icon-editor#208] [General]
As a LabVIEW user, I want a quick way to reorder my layers.

**Current Situation**
Right now, I have to select layers by clicking on them, and then use the up/down arrows. This is a bit cumbersome and not obvious that it's even a possibility (i.e. it's not very "discoverable").

**Possible Solution**

Add some grab handles (visible for the selected layers? or, visible all the time?) to indicate that the layers can be re-positioned.

![image](https://github.com/ni/labview-icon-editor/assets/381432/579dbc01-f39e-46d1-a600-df59d052fe98)

**Comment by svelderrainruiz** [comment]
@gregr-ni please let us know if you have any objections on attempting to implement this.

**Comment by gregr-ni** [comment]
Seems reasonable. The graphics shown in the image seem a bit large but that may have been just for illustration. Will the grab handle be always visible or only show up on hover?

**Comment by j-medland** [comment]
This is a promising idea so we will move it to a discussion to hammer out some technical details before opening for contribution
